docs/src/pages/autodiff.md

```diff
@@ -0,0 +1,152 @@
+# Automatic Differentiation
+
+[Automatic differentiation](https://en.wikipedia.org/wiki/Automatic_differentiation),
+sometimes abbreviated as "autodiff" or simply "AD," refers to the process of computing
+derivatives of arbitrary functions (in the programming sense of the word) in an automated
+way.
+There are two primary styles of automatic differentiation: _forward mode_ (FMAD) and
+_reverse mode_ (RMAD).
+Nabla's implementation is based on the latter.
+
+## What is RMAD?
+
+A comprehensive introduction to AD is out of the scope of this document.
+For that, the reader may be interested in books such as _Evaluating Derivatives_ by
+Griewank and Walther.
+To give a sense of how Nabla works, we'll briefly give a high-level overview of RMAD.
+
+Say you're evaluating a function ``y = f(x)`` with the goal of computing the derivative
+of the output with respect to the input, or, in other words, the _sensitivity_ of the
+output to changes in the input.
+Pick an arbitrary intermediate step in the computation of ``f``, and suppose it has the
+form ``w = g(u, v)`` for some intermediate variables ``u`` and ``v`` and function ``g``.
+We denote the derivative of ``u`` with respect to the input ``x`` as ``\dot{u}``.
+In FMAD, this is typically the quantity of interest.
+In RMAD, we want the derivative of the output ``y`` with respect to (each element of) the
+intermediate variable ``u``, which we'll denote ``\bar{u}``.
+
+[Giles (2008)](https://people.maths.ox.ac.uk/gilesm/files/NA-08-01.pdf) shows us that we
+can compute the sensitivity of ``y`` to changes in ``u`` and ``v`` in reverse mode as
+```math
+\bar{u} = \left( \frac{\partial g}{\partial u} \right)^{\intercal} \bar{w}, \quad
+\bar{v} = \left( \frac{\partial g}{\partial v} \right)^{\intercal} \bar{w}
```

Comment:
this isn't rendered properly when i "view file" ...does github markdown have latex-like maths rendering?

Reply:
Ah yeah, this is MathJax, which GitHub doesn't render in markdown but Documenter-built markdown does.

Reply:
So if you build the docs and view them, it renders properly.